<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>589</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(43, 100, 131);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 70, 90, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QLabel" name="animatedGifLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>741</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>741</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Permanent Marker</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Converts image formats </string>
   </property>
  </widget>
  <widget class="QPushButton" name="fileInButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Permanent Marker</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>input directory of the image sequence</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(130, 130, 130);</string>
   </property>
   <property name="text">
    <string>Input Directory</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changFileFormatButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>510</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Permanent Marker</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>execute image fornat conversion</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(105, 105, 105);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(100, 100, 100, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string>Change Image Format</string>
   </property>
  </widget>
  <widget class="QComboBox" name="imgFileDispayComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>731</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>display the file image sequence</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(90, 90, 90);</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>751</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(112, 112, 112);</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="fileOutButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>390</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Permanent Marker</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>output directory -&gt; output image sequence path</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(130, 130, 130);</string>
   </property>
   <property name="text">
    <string>Output Directory</string>
   </property>
  </widget>
  <widget class="QComboBox" name="imgExtDispayComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>731</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>select the extension for image conversion</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(90, 90, 90);</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSelExt">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Permanent Marker</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>select the extension for image conversion</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 255, 255, 0), stop:1 rgba(255, 255, 255, 0));</string>
   </property>
   <property name="text">
    <string>Select Extension</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>220</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Permanent Marker</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>file sequence display</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
   </property>
   <property name="text">
    <string>File Display</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="animGifButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>450</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Permanent Marker</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>makes animted gif from theinput image sequence</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(130, 130, 130);</string>
   </property>
   <property name="text">
    <string>Make Animted Gif</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
